connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -92,6 +93,7 @@
     private final ExecutorService executor;
     private final Time time;
     private final String workerId;
+    private final String clusterId;
```

Comment:
Removed clusterId from Worker constructor.

Reply:
I'm still keeping clusterId as a class variable in Worker, since we are using clusterId in other method of this class. We are getting clusterId from ConnectUtils.lookupKafkaClusterId inside Worker constructor and keep it as a class variable value.

Reply:
Changed name to kafkaClusterId